tests/integration/test_api_gateway.py

```diff
@@ -450,6 +450,11 @@ def _test_api_gateway_lambda_proxy_integration(
         result = requests.post(url, data=json.dumps({"return_raw_body": body_msg}))
         self.assertEqual(body_msg, to_str(result.content))
 
+        # send message with binary data
+        binary_msg = b"\xff \xaa \x11"
+        result = requests.post(url, data=binary_msg)
+        self.assertEqual("/yCqIBE=", json.loads(to_str(result.content))["body"])
```

Comment:
nit: Additionally, we could assert that `"isBase64Encoded": True` is passed in the Lambda event that contains the base64-encoded data. (we could return the attribute from the Lambda handler, and assert its value here)

Reply:
@whummer thanks for the feedback, will make the follow-up PR as per your suggestion 👍